components/external-link/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { compact, uniq } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+
+/**
+ * Internal dependencies
+ */
+import Dashicon from '../dashicon';
+import './style.scss';
+
+function ExternalLink( { href, children, className, rel = '', ...additionalProps } ) {
+	rel = uniq( compact( [
+		...rel.split( ' ' ),
+		'external',
+		'noreferrer',
+		'noopener',
+	] ) ).join( ' ' );
+	const classes = classnames( 'new-window-icon', className );
```

Comment:
Per recommended CSS Naming guidelines, the default class name should be `components-external-link`

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming